### DISCUSSION TITLE
NPM package missing dist folder (starting with version 37.181.0)

### DISCUSSION THREAD

**Discussion by darthmaim** [renovatebot/renovate#27200] [Request Help] [answered]
### How are you running Renovate?

None

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

The renovate package is missing the dist folder, starting with 37.181.0.

See https://www.npmjs.com/package/renovate/v/37.181.0?activeTab=code (compare with working https://www.npmjs.com/package/renovate/v/37.180.1?activeTab=code)

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by viceice** [comment, marked as answer]
known issue, should be fixed in next release in a few minutes 

https://github.com/renovatebot/renovate/actions/runs/7847634630/job/21417324060

**Reply by darthmaim** [reply]
Confirmed working after 37.181.2 👍
